business_requirement_deliverable/models/business.py

```diff
@@ -227,7 +227,7 @@ def _compute_currency_id(self):
             if br.partner_id and br.pricelist_id.currency_id:
                 br.currency_id = br.pricelist_id.currency_id.id
             else:
-                br.currency_id = self.env.user.company_id.currency_id.id
+                br.currency_id = self.env.company.currency_id.id
```

Comment:
Same here as above

Reply:
It is changed to use the company value from the registry.